src/flb_input_chunk.c

```diff
@@ -18,6 +18,9 @@
  *  limitations under the License.
  */
 
+#define FS_CHUNK_SIZE_DEBUG(op)  {flb_debug("[%d] %s -> fs_chunks_size = %zu", __LINE__, op->name, op->fs_chunks_size);}
```

Comment:
Let's get rid of these for the final version and if you get rid of the macro I left last time I'd be more than grateful.

Reply:
I was intending to leave them in since they would be invaluable in tracking future problems with users.

Reply:
what about making it "trace" level instead of debug ?

- rename from _DEBUG to _TRACE , use flb_trace instead of flb_debug
- please do not exceed lines length, around 90 chars is fine

Reply:
commited!